consensus/view_change.go

```diff
@@ -357,7 +370,7 @@ func (consensus *Consensus) onViewChange(msg *msg_pb.Message) {
 	consensus.viewIDBitmap[recvMsg.ViewID].SetKey(recvMsg.SenderPubkey, true)
 	consensus.getLogger().Debug().
 		Int("have", len(consensus.viewIDSigs[recvMsg.ViewID])).
-		Int64("needed", consensus.Decider.TwoThirdsSignersCount()).
+		Int64("total", consensus.Decider.ParticipantsCount()).
```

Comment:
this is race conidtion , this field prefer rid it

Reply:
The whole onViewChange is locked. I will keep it for debugging purpose.

Reply:
consensus.vcLock.Lock()
	defer consensus.vcLock.Unlock()

Reply:
k, anyway later will replace the map underlying it with a concurrent trie